build/README_DOCKER.md

```diff
@@ -320,10 +320,77 @@ the host in `~/.ipdk/examples`, by executing:
     ipdk [options] examples
 ```
 
-# Section 4: Inner workings
+# Section 4: IPDK CLI inner workings
 
 TODO: Add all specific things about how the IPDK container and cli works.
 
+## IPDK CLI commands
```

Comment:
Please do not document this in the README file, and instead just reference this can be seen when running `ipdk help`. The reason for this is that if you document it here, it will quickly get out of date compared to what is in the ipdk command itself.

Reply:
Ok, agree... Need not find a way to make the help more modular but that shouldn't be to hard😉